dist/index.js

```diff
@@ -16473,6 +16603,26 @@ async function exportSecrets() {
         }
     }
 
+    if (process.env['http_proxy'] || process.env['HTTP_PROXY']) {
+        defaultOptions.agent.http = new HttpProxyAgent({
+            keepAlive: true,
+            keepAliveMsecs: 1000,
+            maxSockets: 256,
+            maxFreeSockets: 256,
+            proxy: process.env['http_proxy'] || process.env['HTTP_PROXY']
+        })
+    }
+
+    if (process.env['https_proxy'] || process.env['HTTPS_PROXY']) {
+        defaultOptions.agent.https = new HttpsProxyAgent({
+            keepAlive: true,
+            keepAliveMsecs: 1000,
+            maxSockets: 256,
+            maxFreeSockets: 256,
+            proxy: process.env['https_proxy'] || process.env['HTTPS_PROXY']
+        })
+    }
+
```

Comment:
We generate the dist/index when releasing new versions its fine to not commit it if you prefer. It often leads to annoying merge conflicts to resolve.